bin/main.rb

```diff
@@ -1,2 +1,88 @@
 #!/usr/bin/env ruby
+require_relative '../lib/board'
+require_relative '../lib/helpers'
+
 puts '"Hello world!"'
```

Comment:
- [ ] You don't need to keep this line in this milestone

Reply:
Thanks for your review. We have fixed the given issues according to review.